Re-sync from v3 source

adamhgriffith-uofu opened the pull request.

mcuma commented:
Thanks Adam, one concern. If we're removing job_params_v33 and replacing it with job_params, we need to create symlink job_params_v33 -> job_params since job_params_v33  is used in the form.yml.erb of the interactive apps. It may be easier to keep job_params_v33 and remove job_params.

adamhgriffith-uofu commented:
Understood. How does the updated PR look?

mcuma commented:
OK, but I am now not seeing the other removed files being removed, like the job_params_v3, etc.

adamhgriffith-uofu commented:
You can ignore the other commits on this branch. I was trying something, then reverted so the [diff](https://github.com/chpc-uofu/ood-ge-apps-v4/pull/4/files) is now just the transition to a symlink.

adamhgriffith-uofu commented:
I kept the other files around since apps like `desktop_basic` referenced some of them. E.g.
```ruby
<% IO.foreach(template_root+"job_params_v3.basic") do |line| %>
<%= line %>
<% end %>   
```

mcuma commented:
Good catch. desktop_basic app is not used anymore, so, I removed it from the original repo. I also removed the old jobs_params files. If you could sync this repo with mine and remove the other job_params files, that's be great.

adamhgriffith-uofu commented:
Done and done.

adamhgriffith-uofu commented:
If all apps need to use `job_params_v33`, mind if I remove the symlink to `job_params` and do a quick find-replace? Then `job_params_v33` and `job_params.basic` can go away, leaving us with a single file `job_params`.

mcuma commented:
Adam, would you mind re-syncing this again with the https://github.com/chpc-uofu/OOD-apps-v3. I have cleaned up the job_params files. I am reluctant to merge the PR because it removes all the node status apps.

adamhgriffith-uofu commented:
@mcuma, re-sync completed.

Change summary
1. Removed *.org, *.dialog, *.log files
2. Under cluster status apps, removed generated files (e.g. lp.html).
3. Removed accidentally copied over openwebui_app and openwebui_vnc_app.
4. Removed extra job_params(_v2/3) files.
5. Created symlink from ./app-templates/job_params_v33 to ./app-templatesjob_params.
